.circleci/config.yml

```diff
@@ -121,6 +121,7 @@ jobs:
       - run:
           name: Wait for node app to start
           command: sleep 5
+#          command: npx wait-on http://localhost:3007
```

Comment:
You can use the command from the helm orb https://github.com/ministryofjustice/offender-categorisation/blob/475a15f6e9ad0c237c8ad413fd8b73c8a913ffd9/.circleci/config.yml#L127-L129